### PR TITLE
docs: offset-syncs topic default location

### PR DESCRIPTION
**Pull request by PaulRMellor**
Signed-off-by: prmellor <pmellor@redhat.com>

**Documentation**
Updates the _MirrorMaker 2_ section of the _Configuring_ guide
Adds information on the default location of the MirrorMaker _offset-syncs_ topic.

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [ ] Write tests
- [ ] Make sure all tests pass
- [x] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md
- [ ] Supply screenshots for visual changes, such as Grafana dashboards

### CODE REVIEW

**Review by mimaison — APPROVED** [review]

File: documentation/modules/configuring/con-config-mirrormaker2-connectors.adoc [hunk 1]
```diff
@@ -40,7 +40,7 @@ The following table describes connector properties and the connectors you config
 |✓
 |
 
-|offset-syncs.topic.location:: The location of the `offset-syncs` topic, which can be the `source` (default) or `target` cluster. If you set the property to `target`, read-only access to the source cluster is granted to MirrorMaker.
+|offset-syncs.topic.location:: The location of the `offset-syncs` topic, which can be the `source` (default) or `target` cluster.
```
Comment (scholzj) [inline documentation/modules/configuring/con-config-mirrormaker2-connectors.adoc:43]: Is the setting `source` and `target`? Or are these the aliases of the source and target clusters?

File: documentation/modules/configuring/con-config-mirrormaker2-replication.adoc [hunk 2]
```diff
@@ -78,14 +78,19 @@ Only MirrorMaker 2.0 can write to remote topics.
...
+* The _checkpoints_ topic maps the last committed offset in the source and target cluster for replicated topic partitions in each consumer group
 
-Offsets for the _checkpoint_ topic are tracked at predetermined intervals through configuration.
+Offsets for the _checkpoints_ topic are tracked at predetermined intervals through configuration.
 Both topics enable replication to be fully restored from the correct offset position on failover.
 
 MirrorMaker 2.0 uses its `MirrorCheckpointConnector` to emit _checkpoints_ for offset tracking.
 
+The location of the _offset-syncs_ topic is the source cluster by default.
+You can use the `offset-syncs.topic.location` connector configuration to change this to the target cluster.
```
Comment (scholzj) [inline documentation/modules/configuring/con-config-mirrormaker2-replication.adoc:90]: Similar to above ... should it be set to
* the value `target` for the target cluster?
* to the alias of the target cluster?

File: documentation/modules/configuring/con-config-mirrormaker2-replication.adoc [hunk 3]
```diff
@@ -78,14 +78,19 @@ Only MirrorMaker 2.0 can write to remote topics.
...
+You need read/write access to the cluster that contains the topic.
+If you change the default, then you'll only need read access on the source cluster.
```
Comment (scholzj) [inline documentation/modules/configuring/con-config-mirrormaker2-replication.adoc:92]: Maybe this scould be more clear?

```suggestion
Using the target cluster as the location of the `offset-sync` topic allows you to use Kafka MirrorMaker 2 even if you have only read access to the source cluster.
```

File: documentation/modules/configuring/con-config-mirrormaker2-connectors.adoc (hunk 1 above)
Comment (mimaison) [reply documentation/modules/configuring/con-config-mirrormaker2-connectors.adoc:43]: Yes this config is set to either `source` or `target`. It is not using the cluster aliases.

File: documentation/modules/configuring/con-config-mirrormaker2-replication.adoc (hunk 2 above)
Comment (scholzj) [reply documentation/modules/configuring/con-config-mirrormaker2-replication.adoc:90]: Following the asnwer above from Mickael => I guess making `target` formatted would make it a bit more clear.

**Review by scholzj — APPROVED** [review]

**Pull request merged**